src/main/java/org/openrewrite/kotlin/KotlinParser.java

```diff
@@ -251,6 +254,11 @@ public Builder logCompilationWarningsAndErrors(boolean logCompilationWarningsAnd
             return this;
         }
 
+        public Builder isKotlinScript(boolean isKotlinScript) {
```

Comment:
It seems like currently the only reason for this option is that the test harness doesn't specify a file name when invoking the parser. I was just wondering if we actually need a separate parser for .kts files.